File: 1-two-sum/1-two-sum.py
class Solution:
    def twoSum(self, nums: List[int], target: int) -> List[int]:
        ans = {}
        for i in range(len(nums)):
            if ans.__contains__(target - nums[i]):
                return [i, ans[target-nums[i]]]
            else:
                ans[nums[i]] = i
        
        

File: 1-two-sum/1-two-sum.java
class Solution {
    public int[] twoSum(int[] nums, int target) {
        int[] sol = new int[2];
        int temp = 0;
        for(int i = 0; i< nums.length; i++){
            for (int j = 0; j< nums.length; j++){
                if (nums[i] + nums[j] == target & i!= j){
                    sol[0] = i;
                    sol[1] = j;
                }
                
            }
        }
        return sol;
        
    }
}